.ko.yaml

```diff
@@ -0,0 +1 @@
+defaultBaseImage: artprod.dev.bloomberg.com/ds/base/ubuntu:18.04.20180515.1
```

Comment:
this doesn't look like it should be checked in.

Reply:
oops~ Will remove.